packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksContent.tsx

```diff
@@ -41,20 +198,51 @@ const baseRenderLeaf = (props: RenderLeafProps, modifiers: ModifiersStore) => {
   return <span {...props.attributes}>{wrappedChildren}</span>;
 };
 
-const baseRenderElement = (props: RenderElementProps, blocks: BlocksStore) => {
+type baseRenderElementProps = {
```

Comment:
I think for simple types for this interface is better since it can be extended

Reply:
we can intersect the type, also I found this - https://stackoverflow.com/a/41385149